### PR TITLE
[Keyboard] Add earth_rover keyboard

### PR DESCRIPTION
**Pull request by bigwheel**
Add new keyboard [earth_rover](https://github.com/bigwheel/earth_rover).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by bigwheel** [comment]
I can't understand this travis error. Could anyone direct how to fix this ?

**Comment by drashna** [comment]
Did you mean to target the develop branch here?  I think there are some bugs with the develop branch right now.

**Review by drashna — APPROVED** [review]

**Comment by bigwheel** [comment]
Thanks! @drashna 
https://docs.qmk.fm/#/pr_checklist?id=core-prs

I read this

> must now target develop branch, which will subsequently be merged back to master on the breaking changes timeline

but I didn't read section title

> Core PRs

I fix PR target now!

**Review by drashna — APPROVED** [review]

File: keyboards/earth_rover/config.h [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+//#    define RGBLIGHT_EFFECT_RGB_TEST
+//#    define RGBLIGHT_EFFECT_ALTERNATING
+/*== customize breathing effect ==*/
+/*==== (DEFAULT) use fixed table instead of exp() and sin() ====*/
+//#    define RGBLIGHT_BREATHE_TABLE_SIZE 256      // 256(default) or 128 or 64
+/*==== use exp() and sin() ====*/
+//#    define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+//#    define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+//#endif
+
```
Comment (noroadsleft) [inline keyboards/earth_rover/config.h:92]:
```suggestion
```

File: keyboards/earth_rover/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    [_BASE] = LAYOUT(
+              KC_A, KC_B,
+        KC_C, KC_D, KC_E, KC_F,
+        KC_G, KC_H, KC_I, KC_J,
+        KC_K, KC_L, KC_M, KC_N
+    )
+};
+// clang-format on
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) { return true; }
```
Comment (noroadsleft) [inline keyboards/earth_rover/keymaps/default/keymap.c:33]:
```suggestion
```

File: keyboards/earth_rover/readme.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# earth_rover
+
+![earth_rover](https://pbs.twimg.com/media/Eqh29nYUUAA-9NE?format=jpg)
+
+*A short description of the keyboard/project*
```
Comment (noroadsleft) [inline keyboards/earth_rover/readme.md:5]:
```suggestion
A prism-shaped keypad assembled from MX-compatible single-key PCBs.
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/earth_rover/readme.md (hunk 3 above)
Comment (bigwheel) [reply keyboards/earth_rover/readme.md]: I think this description is for another keyboard. I update this line with another words.

File: keyboards/earth_rover/readme.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# earth_rover
+
+![earth_rover](https://pbs.twimg.com/media/Eqh29nYUUAA-9NE?format=jpg)
+
+*Mechanical warm, or just cubic macro pad*
```
Comment (noroadsleft) [inline keyboards/earth_rover/readme.md:5]: The italics are just to call attention to this line before it's filled out; the line isn't intended to be kept italicized.

```suggestion
Mechanical warm, or just cubic macro pad.
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**